### DISCUSSION TITLE
v2.0.1 pypi release

### DISCUSSION THREAD

**Discussion by FasterSpeeding** [aio-libs-abandoned/aioredis-py#1250] [Q&A] [answered]
Saw that v2.0.1 was released on the repo a couple of days ago but that doesn't seem to have made it to PyPi, is there a reason for this or was it just missed or something?

**Comment by Andrew-Chen-Wang** [comment, marked as answer]
Omg sorry! It should now be published! Thank you for informing me of this; the PR didn't include a tag (not used to that system). https://github.com/aio-libs/aioredis-py/runs/4644933958?check_suite_focus=true
